### ISSUE TITLE
Setting padding to 0 doesn't seem to work

### ISSUE THREAD

**Issue by wren** [Pocco81/true-zen.nvim#107]
Hi! Congrats on the plugin rewrite! I know it can be a daunting undertaking to throw everything out and start anew.

I'm having some trouble with the new padding setting, though. When I set any given side to 0, there's always still a window there, and it always has a minimum width/height of 1. This is in ataraxis mode, by the way, but I'm not sure if that's relevant (haven't tested with the other modes).

**Comment by ShadmanAhm3d** [comment]
Hey did you find any solution to this? @wren

**Comment by wren** [comment]
@ShadmanAhm3d No, I didn't. Sorry! Unfortunately, this plugin seems to have been abandoned. 😞
